src/styles/blocks/main-cards.scss

```diff
@@ -0,0 +1,24 @@
+.main {
+  display: flex;
+  flex-direction: column;
+
+  @media (min-width:1024px) {
+    padding: 50px 40px 72px;
+  }
+  @media (min-width:1200px) {
+    padding: 50px 128px 72px;
+  }
```

Comment:
I guess we don't media in this task,  you can just center your card container using margin: 0 auto rule